### DISCUSSION TITLE
Adding a port with a release only build cmake tool

### DISCUSSION THREAD

**Discussion by gauntl3t12** [microsoft/vcpkg#36314] [Q&A] [answered]
I'm trying to add a port for the [corrosion](https://corrosion-rs.github.io/corrosion/introduction.html) CMake tool. It works but gets god almighty warnings. I'm curious if there are any known solutions/workarounds for similar tools.

## Issues
1. Corrosion is only intended to be built in Release mode it seems (otherwise it won't install)
2. The tool when installed, drops absolute paths in

**Comment by Osyotr** [comment, marked as answer]
IIUC it's not a tool but a bunch of cmake scripts. Check out cmakerc port https://github.com/microsoft/vcpkg/blob/a1d682b5aa5385c849a2beb0e1203757b9625d41/ports/cmakerc/portfile.cmake
